src/GraphQL.Tests/Utilities/SchemaBuilderExecutionTests.cs

```diff
@@ -320,6 +322,33 @@ public void can_use_context_source_usercontext_with_params()
 
             result.ShouldBe(serializedExpectedResult);
         }
+
+        [Fact]
+        public void can_use_DateTimeOffset_type(){
+            var schema = Schema.For(@"
+                input DateTimeOffsetInput{
+                    value: Date
+                }
+                type Query {
+                  five(model: DateTimeOffsetInput): Date
+                }
+            ", _=>
+            {
+                _.Types.Include<ParametersType>();
+            });
+
+            var utcNow = DateTimeOffset.UtcNow;
+
+            var result = schema.Execute(_ =>
+            {
+                _.Query = $"{{ five(model:{{ value:\"{utcNow}\"}}) }}";
+            });
+
+            var expectedResult = CreateQueryResult($"{{ 'five': \"{utcNow.AddDays(1).ToString("yyyy-MM-ddTHH:mm:ssZ")}\" }}");
+            var serializedExpectedResult = Writer.Write(expectedResult);
+
+            result.ShouldBe(serializedExpectedResult);
+        }
```

Comment:
Can you add this to its own test file?  This test is not super related to the other ones in this class/file.

Reply:
Sure, extracted to separate file.